src/configuration.ts

```diff
@@ -19,6 +19,7 @@ import * as fs from "fs";
 import * as path from "path";
 import * as findup from "findup-sync";
 
+const PACKAGE_DEPRICATION_MSG = "configuration of tslint via package.json has been deprecated, please start using the standard tslint.json"
```

Comment:
* `DEPRECATION` instead of `DEPRICATION`
* `TSLint` instead of `tslint` (I know we use these sort of interchangeably)
* semicolon at the end of the line
* `Configuration` instead of `configuration`.
* Could we also include this URL? http://palantir.github.io/tslint/usage/tslint-json/